src/makeCalculator.js

```diff
@@ -36,8 +36,47 @@
  *
  * @return {object}
  */
+
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function(number) {
+      calculator.result += number;
+
+      return calculator;
```

Comment:
better to use
```suggestion
      return this;
```

Reply:
if replaced then this error pops up:
● sum › 'operate' should support a chain call 

NOT better to use